pypreprocessor/__init__.py

```diff
@@ -100,7 +100,9 @@ def __if(self):
 
     # evaluate
     def lexer(self, line):
-    # return values are (squelch, metadata)
+        # strip any and all leading whitespace characters
+        line = line.lstrip()
+        # return values are (squelch, metadata)
```

Comment:
For the sake of [docstrings](https://www.python.org/dev/peps/pep-0257/), I would suggest the other way around:

    """Returns (squelch, metadata)."""
    # strip leading whitespace characters for compatibility with the C preprocessor.
    line = line.lstrip()